public/components/overview/home.tsx

```diff
@@ -3,159 +3,272 @@
 
-const navigateToApp = (appId: string, path: string) => {
+export const navigateToApp = (appId: string, path: string) => {
   coreRefs?.application!.navigateToApp(appId, {
     path: `${path}`,
   });
 };
 
-export type AppAnalyticsCoreDeps = TraceAnalyticsCoreDeps;
+coreRefs.contentManagement?.registerContentProvider({
+  id: 'custom_content',
+  getContent: () => ({
+    id: 'custom_content',
+    kind: 'custom',
+    order: 1500,
+    render: () =>
+      wrapper.dashboardSelected ? (
+        <DashboardControls
+          dashboardTitle={dashboardTitle}
+          dashboardId={dashboardId}
+          startDate={startDate}
+          setStartDate={setStartDate}
+          endDate={endDate}
+          setEndDate={setEndDate}
+          showFlyout={showFlyout}
+        />
+      ) : (
+        <AddDashboardCallout showFlyout={showFlyout} navigateToApp={navigateToApp} />
+      ),
+  }),
+  getTargetArea: () => HOME_CONTENT_AREAS.SELECTOR,
+});
 
-interface HomeProps extends RouteComponentProps, AppAnalyticsCoreDeps {
-  parentBreadcrumbs: ChromeBreadcrumb[];
+export interface DashboardDictionary {
+  [key: string]: {
+    value: string;
+    label: string;
+    startDate: string;
+    endDate: string;
+  };
 }
 
-const HomeContent = ({ alertsPluginExists, anomalyPluginExists }) => (
-  <div>
-    <EuiSpacer size="l" />
-    <EuiText>
-      <h2>Observability overview</h2>
-    </EuiText>
-    <EuiSpacer size="l" />
-    <EuiFlexGroup gutterSize="l">
-      {[
-        {
-          id: gettingStartedURL,
-          title: 'Get started collecting and analyzing data.',
-          description: 'getting started guide',
-        },
-        {
-          id: discoverURL,
-          title: 'Uncover insights with raw data exploration.',
-          description: 'with Discover',
-          path: 'discover',
-        },
-        {
-          id: metricsURL,
-          title: 'Transform logs into actionable visualizations with metrics extraction.',
-          description: 'with Metrics',
-        },
-        {
-          id: tracesURL,
-          title: 'Unveil performance bottlenecks with event flow visualization.',
-          description: 'with Traces',
+export const Home = ({ ..._props }: HomeProps) => {
+  const homepage = coreRefs.contentManagement?.renderPage(HOME_PAGE_ID);
+  const [_, setIsRegistered] = useState(false);
+  const [dashboards, setDashboards] = useState<DashboardDictionary>({});
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
+  [startDate, setStartDate] = useState(moment().toISOString());
+  [endDate, setEndDate] = useState(moment().toISOString());
+  [dashboardTitle, setDashboardTitle] = useState('');
+  [dashboardId, setDashboardId] = useState('');
+
+  showFlyout = () => setIsFlyoutVisible(true);
+
+  const registerCards = async () => {
+    let alertsPluginExists = false;
+    let anomalyPluginExists = false;
+
+    try {
+      const response = await fetch('../api/status', {
```

Comment:
Not sure, but can we use `coreRef.http.get('api/status')` here. Also, can move api/status to constants. Using fetch is generally not recommended. Even though used in older code bases

Reply:
coreRef call not working will attempt change again in follow up PR.

Replaced with import of
API_STATUS_URL
located in the constant folder.

Reply:
updated in my changes